easybuild/easyconfigs/d/deepTools/deepTools-3.5.2-foss-2022a.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'deepTools'
+version = '3.5.2'
+
+homepage = 'https://deeptools.readthedocs.io/'
+description = """deepTools is a suite of python tools particularly developed for the efficient analysis of
+ high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('plotly.py', '5.12.0'),
+    ('Pysam', '0.19.1'),
+    ('pyBigWig', '0.3.18'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('py2bit', '0.3.0', {
+        'checksums': ['450555c40cba66957ac8c9a4b6afb625fb34c4bb41638de78c87661ff8b682ef'],
+    }),
+    ('deeptoolsintervals', '0.1.9', {
+        'checksums': ['7d94c36fd2b6f10d8b99e536d2672e8228971f1fc810497d33527bba2c40d4f6'],
+    }),
+    ('numpydoc', '1.5.0', {
+        'checksums': ['b0db7b75a32367a0e25c23b397842c65e344a1206524d16c8069f0a1c91b5f4c'],
+    }),
+    (name, version, {
+        'checksums': ['9367f9037b1822b7d69d5abaf47ca25bf0e5dc4cb8be85bd55b6f63c90781941'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Let's add some sanity check commands:
```suggestion
}

sanity_check_commands = [
    "bamCompare --help",
    "multiBamSummary --help",
    "plotHeatmap --help",
]